Form/Extension/DateTypeExtension.php

```diff
@@ -22,7 +22,9 @@ public function finishView(FormView $view, FormInterface $form, array $options)
         if ('single_text' === $options['widget'] && isset($options['datepicker'])) {
             $view->vars['datepicker'] = $options['datepicker'];
         }
-
```

Comment:
Should be a space between the two if statements. I think I might have commented on the wrong line